app/src/app/pages/genre/genre.component.html

```diff
@@ -12,6 +14,6 @@
     </div>
   </div>
   <span class="artworks-amount">{{ genre?.count }}</span>
-  <span class="ml-1 slider-heading" i18n="Artworks label@@artworks">Artworks</span>
+  <span class="ml-1 slider-heading" *ngIf="!idDoesNotExist" i18n="Artworks label@@artworks">Artworks</span>
```

Comment:
This should not be necessary when whole content is hidden if idDoesNotExist is true